### ISSUE TITLE
Mark this repository as unmaintained

### ISSUE THREAD

**Issue by andymckay** [actions/delete-package-versions#36]
Sadly we haven't got the time to maintain this repository as an official GitHub Action. In the last year or two since we originally published this, a large number of really good community Actions have appeared on the Marketplace. There's likely some good alternatives that are maintained that we could recommend instead.

We'd like to mark this repository as unmaintained and make it clear to users what their expectations should be. When its archived, **it will continue to work for all existing Actions users, but won't receive any updates**.

People are of course free to fork this repository and maintain those forks if they like.

To do this we'll:

* [ ] Remove it from the Marketplace
* [ ] Remove from any official GitHub documentation.
* [ ] Change the README.md to clearly reflect its new state.
* [ ] Update the README.md to point to other community Actions.
* [ ] Archive this repository, putting it read-only mode.

**Comment by Ocramius** [comment]
Still in https://docs.github.com/en/packages/learn-github-packages/deleting-and-restoring-a-package#automate-package-version-deletion-with-github-actions BTW (doesn't seem like there are any alternatives to this action right now)

**Comment by andymckay** [comment]
Thank you @Ocramius

**Comment by jaschaio** [comment]
@andymckay whats a good alternative?

**Comment by peter-dolkens** [comment]
Still looking for alternatives.

We've used our included enterprise storage quota now, but GitHub offers no retention policy functionality, and links to an unmaintained action that is lacking in basic features (delete older than X days, or delete older than version Y, for example) as a solution.

GitHub Actions has enjoyed great adoption rates, but this feels like a big miss, which could hurt those early adopters.

**Comment by peter-dolkens** [comment]
@jaschaio - I found https://github.com/SmartsquareGmbH/delete-old-packages

**Comment by trent-j** [comment]
👋 I was the original author of this action and admittedly I have neglected it due to my EPD leads deciding it wasn't a high priority. We (the Packages team) have recently released a new unified REST API to replace the current GraphQL API which is currently what this action is using. 

We are also in the process of migrating docker v1 packages to our new Container Registry. During this work I got the go ahead to retro fit this action to use the new REST API and add some of the community requested features. If I were to make a new `v2` version of this action with support for new features would that change the decision of marking this as unmaintained and removing it from the docs?

I can let my EPD leads know that we need to allocate some time to maintaining this action if so.
